django_elasticsearch_dsl/documents.py

```diff
@@ -143,7 +144,14 @@ def to_field(cls, field_name, model_field):
             )
 
     def bulk(self, actions, **kwargs):
-        return bulk(client=self._get_connection(), actions=actions, **kwargs)
+        response = bulk(client=self._get_connection(), actions=actions, **kwargs)
+        # send post index signal
+        post_index.send(
+            sender=self.__class__,
+            instance=self,
```

Comment:
I think you should pass the actions as well to the signal. Without it, it is quite hard to determine which object is indexed to Elasticsaerch.